chains/v13/chains.json

```diff
@@ -2819,6 +2838,20 @@
                     "existentialDeposit": "10000",
                     "transfersEnabled": true
                 }
+            },
+            {
+                "assetId": 14,
+                "symbol": "DAI",
```

Comment:
not sure also about this one as it is a ( portal ) coin, don't think we have decided sth about it:
<img width="510" alt="image" src="https://github.com/novasamatech/nova-utils/assets/16337578/d58f8bfe-959c-423c-b464-aa1b58badd43">

Reply:
it can be send via XCM:
https://acala.subscan.io/xcm_message/polkadot-76d7db35c5cd8d3c783376acf4ee5245ab3ee8d7
looks valid